Resources/config/services.xml

```diff
@@ -17,12 +17,14 @@
     </parameters>
 
     <services>
-        <service id="bg_object_routing.adapter.symfony_21" class="%bg_object_routing.adapter.symfony_21.class%">
+        <service id="bg_object_routing.adapter.symfony_22" class="JMS\ObjectRouting\Symfony\Symfony22Adapter" public="false">
```

Comment:
I removed the parameter as it was my understanding that this approach was a mistake during the early days of Symfony that has meanwhile been discouraged.

But I will change it later if you'd like to keep it.